src/contentChunk.js

```diff
@@ -85,13 +85,13 @@ const removeBodyById = (id) => {
   })
 }
 
-exports.promisesToRemoveAllTransactionBodies = (tx) => {
+export const promisesToRemoveAllTransactionBodies = (tx) => {
```

Comment:
This only removes the request and response bodies. Shouldn't we also remove the orchestration and route response and request bodies?

Reply:
Ive only implemented the removal of the main request/response body to get the basics functionality working while the orchestration/route ticket was still being worked on. Ill add in the logic for the others as well